rules/what-is-figma/rule.md

```diff
@@ -1,7 +1,7 @@
 ---
 type: rule
-title: Do you know what Figma is?
-uri: what-is-figma
+title: Do you know Figma's top 5 uses ?
+uri: figmas-top-5-uses
```

Comment:
1. When you change a URL, you also need to update the folder name: https://github.com/SSWConsulting/SSW.Rules.Content/wiki/How-to-Rename-Rules

2. I don't like "top-5" in the URI. If we get another use and change to 6 it will make the work harder.
I would change to "top-figma-uses" only